instrumentation/spring-rabbit/src/test/java/brave/spring/rabbit/ITSpringRabbitTracing.java

```diff
@@ -150,4 +152,16 @@
     awaitMessageConsumed();
     // reporter rules verify nothing was reported
   }
+
+  @Test public void batchConsumerTest() {
+    produceUntracedMessage(exchange_batch.getName(), binding_batch);
```

Comment:
please add a test for two messages in all combinations as it shows a problem to solve later

non traced that listener is a new trace
first traced that listener continues that trace
two traced that listener continues first  trace but TODO we aren't handling the second.
last trace that listener is new trace and TODO about the second

Reply:
fixed at last commit.